app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -165,15 +172,20 @@ ControllerContainer::CreateController(const int32_t aControllerIndex, const int3
     m.SetUpModelsGroup(aModelIndex);
     CreationContextPtr create = m.context.lock();
     controller.transform = Transform::Create(create);
+    controller.pointerTransform = Transform::Create(create);
     if ((m.models.size() >= aModelIndex) && m.models[aModelIndex]) {
       controller.transform->AddNode(m.models[aModelIndex]);
       if (m.pointerModel) {
-        controller.transform->AddNode(m.pointerModel);
+        controller.pointerTransform->AddNode(m.pointerModel);
       }
       if (m.root) {
         m.root->AddNode(controller.transform);
         m.root->ToggleChild(*controller.transform, false);
       }
+      if (m.pointerRoot) {
+        m.pointerRoot->AddNode(controller.pointerTransform);
+        m.pointerRoot->ToggleChild(*m.pointerModel, false);
```

Comment:
This should be `m.pointerRoot->ToggleChild(*controller.pointerTransform, false);`

This why there is a visible point at the users feet.